lab-jaren/lib/parse-message.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.nickCommand = function(message, socket) {
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`you are now ${socket.username}\n`);
+};
+
+exports.dmCommand = function(message, array) {
+  let toUsername = message.split(' ')[1];
+  let content = message.split(' ').slice(2).join(' ').trim();
+  array.forEach(user => {
+    if (user.username === toUsername)
+      user.write(content + '\n');
```

Comment:
Get in the habit of using ES6 interpolation for concatenating strings. The above would be: 
```
`${content}\n`
```